# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../libsafe/destdir.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- libsafe-2.0-16/src/Makefile.vanilla	2007-08-17 14:40:04.000000000 +0200
+++ libsafe-2.0-16/src/Makefile	2007-08-17 14:42:27.000000000 +0200
@@ -118,8 +118,7 @@
 		 rm -f $(PRELOAD) ;\
 		fi; \
 
-		$(INSTALL) $(LIBNAME) $(LIBDIR) 
-		/sbin/ldconfig
+		$(INSTALL) $(LIBNAME) $(DESTDIR)$(LIBDIR) 
 
 		 @if [ -s $(PRELOAD).tmp ] ; then \
 		  cp -fp $(PRELOAD).tmp $(PRELOAD);\
@@ -136,12 +135,12 @@
 
 # If you have man pages in some other directory  add it here.
 
-		@if [  -d $(MANDIR) ];  then \
-		 echo "$(INSTALL) $(MANNAME) $(MANDIR)"; \
-		 $(INSTALL) $(MANNAME) $(MANDIR); \
-		elif [  -d /usr/share/man/man8 ];  then \
-		  echo "$(INSTALL) $(MANNAME) /usr/share/man/man8"; \
-		  $(INSTALL) $(MANNAME) /usr/share/man/man8; \
+		@if [  -d $(DESTDIR)$(MANDIR) ];  then \
+		 echo "$(INSTALL) $(MANNAME) $(DESTDIR)$(MANDIR)"; \
+		 $(INSTALL) $(MANNAME) $(DESTDIR)$(MANDIR); \
+		elif [  -d $(DESTDIR)/usr/share/man/man8 ];  then \
+		  echo "$(INSTALL) $(MANNAME) $(DESTDIR)/usr/share/man/man8"; \
+		  $(INSTALL) $(MANNAME) $(DESTDIR)/usr/share/man/man8; \
 		else echo "***** Error: Cannot find man directories"; \
 		fi
 
